compiler/ast/asdl_rs.py

```diff
@@ -649,7 +649,7 @@ def write_ast_def(mod, typeinfo, f):
         #![allow(clippy::derive_partial_eq_without_eq)]
         
         pub use crate::constant::*;
-        pub use crate::Location;
+        pub use ruff_text_size::{TextSize, TextRange};
```

Comment:
maybe `rustpython_parser::text_size` if the only direct dependency is rustpython-parser?

Reply:
We could re-export the types for crates that only depend on `rustpython_parser`. Having the type in a separate crate does have advantages for us, for example, ruff_diagnostics no longer depends on `ruff_text_size`, which helps speeding up our compile times.

Reply:
I wasn't sure it is only used to replace Location or have more usage in ruff. Thank you!